app/models/certificate.rb

```diff
@@ -16,6 +16,13 @@ def initialize(x509_cert)
 
   def_delegators :@cert_policies, :allowed_by_policy?, :critical_policies_recognized?
 
+  def auth_cert?
+    @x509_cert.extensions.each do |ext|
+      return true if ext.to_s =~ /^extendedKeyUsage.+1\.3\.6\.1\.5\.2\.3\.4/
+    end
+    false
```

Comment:
1. Is that specific string of numbers meaningful?
2. I just realized we could maybe clean this up with:

```
@x509_cert.extensions.any? do |ext|
  ext.to_s =~ /^extendedKeyUsage.+1\.3\.6\.1\.5\.2\.3\.4/
end
```

and we could probably make the regex a constant

Reply:
nice!